watsapp/generate_auth.py

```diff
@@ -0,0 +1,9 @@
+import base64
+
+api_key = '6fc9d82d'  # Your API key from Vonage
```

Comment:
please remove all keys from commits on github. There are multiple ways of managing keys, from creating a [template file](https://github.com/miguelgfierro/ai_projects/blob/main/Intro_to_Chatbots/secretos_template.py) and then ignoring the secret file, to use python dot env. 

You don't have to do this, but in the industry the way to manage secrets is by something called service principal